### PR TITLE
[conflict] MusicController

### PR DESCRIPTION
**Pull request by Ferr013**
Hope this version is better. I'm sorry for the work I' m throwing at you.
Now I 'll try to make the Volume menu work

### CODE REVIEW

**Comment by Sommerbo** [comment]
This supersedes closed PR #622

**Comment by vogonistic** [comment]
The whitespace needs to be fixed. Does anything call or add this, or is it created so that someone can add it later?

File: Assets/Scripts/Controllers/MusicController.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+using UnityEngine;
+using System.Collections;
+
+[RequireComponent (typeof (AudioSource))]
+public class MusicController : MonoBehaviour
+{
+    AudioSource backgroundMusic;
+
+	void Awake ()
```
Comment (Mizipzor) [inline Assets/Scripts/Controllers/MusicController.cs]: Tab.

File: Assets/Scripts/Controllers/MusicController.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        backgroundMusic = GetComponent<AudioSource>();
+        AudioClip ac = Resources.Load<AudioClip>("Music/SpaceTheme");
+        backgroundMusic.clip = ac;
+        backgroundMusic.loop = true;
+        backgroundMusic.Play();
+
+	}
```
Comment (Mizipzor) [inline Assets/Scripts/Controllers/MusicController.cs]: Tab.

**Comment by Ferr013** [comment]
@vogonistic I'm sorry, but I haven't understood what the problem is

**Comment by NogginBops** [comment]
4 spaces instead of tabs. Should be a setting in your IDE.

**Comment by Ferr013** [comment]
Excuse me for the mess I'm making, if also this version doesn't work I'll upload only the music in another PR after closing this one.

**Comment by Mizipzor** [comment]
@Ferr013 Regardless, uploading the music first and creating the controller second absolutely makes sense. Having many small pull requests is actually less work, not more. It also opens up for someone else to create the controller with your music if you give up. :)

**Comment by vogonistic** [comment]
@Ferr013 Something went wrong with this PR. Would you like me to help you clean it up, or do you want to create a new PR from a clean branch and just add the Music and MusicController?

I'm adding `[conflict]` to the title. If you want to keep working with this PR, just remove it from the title when it's fixed. Otherwise, you can close this and open another.

Let us know if you need help with anything. Thanks!

**Comment by Ferr013** [comment]
@vogonistic I'd really appreciate if you could help me clean it up! Tell me what is the best thing to do and I'll try it, or if you can modify this PR and fix it feel free to do it, I'll try to excercise a little bit more in the meantime! Thank you

**Comment by vogonistic** [comment]
The problem is that you did your work on master. I'll let you know when I have it sorted out.

**Comment by vogonistic** [comment]
Apparently I don't understand the sound system enough to fix this on a short amount of time. Here is what I recommend. Pull in the latest master from here, create a new branch called `MusicController` or something like that. Assuming you use git in bash or a shell, type:
`git cherry-pick 9b2f661cd189bb823a567f0ce085e3046ea9c51d`

It'll pick up the important commit that started all of this and put it on that branch and you can keep working on it. I highly encourage you to find a way to play the music without altering the scene because we get merge conflicts there all the time.

I'm closing this PR, create a new one from the new branch.

**Pull request closed without merging**